content/en/llm_observability/setup/sdk/_index.md

```diff
@@ -387,6 +387,10 @@ The `LLMObs.annotate()` method accepts the following arguments:
 : optional - _JSON serializable type or list of dictionaries_ 
 <br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans). **Note**: Retrieval spans are a special case and require a string or a dictionary (or a list of dictionaries) with this format: `{"text": "...", "name": "...", "score": float, "id": "..."}`.
 
+`prompt` 
+: optional - _dictionary_ 
+<br />A dictionary represents the prompt used for an LLM call in the following form: `{"template": "...", "id": "...", "version": "...", "variables": dict[str, str]}`.  **Note**: This argument is only applicable to LLM spans.
```

Comment:
```suggestion
<br />A dictionary represents the prompt used for an LLM call in the following form: `{"template": "...", "id": "...", "version": "...", "variables": {"variable_1": "...", ...}}`.  **Note**: This argument is only applicable to LLM spans.
```
Let's also mention that you can use the `Prompt` helper class to create this argument here :)